all-a-board/src/components/Main/VoteBox.jsx

```diff
@@ -1,13 +1,19 @@
 import './VoteBox.css'
 import { useState } from "react";
-import { reviewVote } from '../../axios'
-export default function VoteBox({currentVotes, review_id}) {
+import { reviewVote, commentVote } from '../../axios'
+
+export default function VoteBox({currentVotes, review_id, comment_id}) {
     const [votes, setVotes] = useState(currentVotes)
```

Comment:
I would set votes to 0 by default rather than use the current votes, then in you h3 you could just add currentVotes and stateVotes, its just that if for whatever reason the votes being passed in props changes it won't update the state